docs/content/any/examples/gitclub.md

```diff
@@ -0,0 +1,9 @@
+---
+title: GitClub
+description: "GitClub is a complete example application with backends
+using Oso for authorization in multiple languages."
+href: https://github.com/osohq/oso
```

Comment:
What's the goal here? Should this point to the GitClub repo (maybe for the currently selected language)?

Reply:
Yeah this should point to gitclub, thanks for catching.